tools/cli/wskaction.py

```diff
@@ -170,7 +174,10 @@ def getExec(self, args, props):
             exe = self.getActionExec(args, props, pipeAction)
         elif args.artifact is not None and os.path.isfile(args.artifact):
             contents = open(args.artifact, 'rb').read()
-            if args.artifact.endswith('.swift'):
+            if args.kind in ['swift:3','swift:3.0','swift:3.0.0']:
```

Comment:
I'm fine changing the syntax to something more consistent. @sjfink , what do you think?